frontend/src/metabase/lib/formatting/link.ts

```diff
@@ -76,7 +80,7 @@ function getValueAndColumnForColumnName(
   columnName: string,
 ) {
   const name = columnName.toLowerCase();
-  const dataSources: any[] = [
+  const dataSources: [string, Record<string, { value: Value }>][] = [
```

Comment:
can it be not string, but more specific? like `column | filter ...`?

Reply:
I haven't found any useful type for this.